lib/class-wp-block-type.php

```diff
@@ -149,6 +149,16 @@ public function prepare_attributes_for_render( $attributes ) {
 			$prepared_attributes[ $attribute_name ] = $value;
 		}
 
+		/**
+		* Filter to allow plugins to override Block attributes.
+		* 
+		* @since 3.5.0
+		* 
+		* @param array $prepared_attributes Block attributes.
+		* @param WP_Block_Type $this Block object which attributes are filtered.
```

Comment:
For the documentation parser it rename the variable here in the docblock to `$block_type`.

Also params should be aligned.